cpp/tests/hash_map/multimap_test.cu

```diff
@@ -91,12 +96,3 @@ TYPED_TEST(MultimapTest, InitialState)
   auto end   = this->the_map->end();
   EXPECT_NE(begin, end);
 }
-
-TYPED_TEST(MultimapTest, CheckUnusedValues)
```

Comment:
why is this test removed?

Reply:
To remove the dependence on managed memory. It was accessing it from the host.

Reply:
Is there a way to add alternate test for this ?

Reply:
It's not necessary.